release-notes/opensearch-performance-analyzer.release-notes-2.3.md

```diff
@@ -0,0 +1,9 @@
+## Version 2.3 Release Notes
+
+Compatible with OpenSearch 2.3
+
+
+### Maintenance
+
+* Upgrade netty version to 4.1.79 ([#249](https://github.com/opensearch-project/performance-analyzer/pull/249))
+* Jackson version bump ([#247](https://github.com/opensearch-project/performance-analyzer/pull/247))
```

Comment:
These were part of 2.2 release, not needed in 2.3? Can you update it accordingly?

Reply:
My bad. You can keep this as it is.